mint/tranformedData.ts

```diff
@@ -9,7 +9,7 @@ export function useTransformation(props: TransformationProps) {
     const evaluateFormula = (formula: string) => {
         const interpolatedFormula = formula.replace(/\{([^}]+)\}/g,
             (match) => props.settings[match.replace(/\{|\}/g, "")]);
-       return Math.round(evaluate(interpolatedFormula));
```

Comment:
Moved this out, as you previously suggested for the graphs it can be controlled in the plotly config, and for the table might want it to be specified on a column by column basis.